example/main.go

```diff
@@ -2,14 +2,31 @@ package main
 
 import (
 	"fmt"
+	"os"
+	"path/filepath"
+	"runtime"
 
-	"github.com/getlantern/pacon"
+	"github.com/getlantern/pac"
 )
 
 func main() {
-	pacon.PacOn("localhost:12345/pac")
-	fmt.Println("proxy set, any key to continue...")
+	pac.SetHelperPath(absPath("./pac-cmd"))
+	if runtime.GOOS == "darwin" {
+		pac.SetIconPathOnOSX(absPath("icon.png"))
```

Comment:
I just noticed that the whole lib is not thread safe. It may leave pac enabled but pac url to empty if the execution of pac.On() and pac.Off() interleaves. We'd better just leave it in documentation.